### PR TITLE
Added io language

### PR DESCRIPTION
**Pull request by Noy**

### CODE REVIEW

**Comment by nloveladyallen** [comment]
Doesn't work. The fifth line should read "Buzz" but instead reads "FizzBuzz".

**Comment by Noy** [comment]
my bad haha, fixed.

File: io.io [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#io language http://iolanguage.org/
+
+FizzBuzz := method(a,
+	if (a % 15 == 0, writeln("FizzBuzz"), 
+		if (a % 3 == 0, writeln("Fizz"), 
+			if (a % 5 == 0, writeln("Buzz"), 
+	writeln(a)
```
Comment (nloveladyallen) [inline io.io]: Is this the indentation style for io? Seems like `writeln(a)` should be indented a level further than the conditions above it.

Comment (Noy) [reply io.io]: You're right actually, lemme fix that

**Pull request merged**